exchanges/okcoin/okcoin_wrapper.go

```diff
@@ -29,39 +28,36 @@ import (
 )
 
 // GetDefaultConfig returns a default exchange config
-func (o *OKCoin) GetDefaultConfig(ctx context.Context) (*config.Exchange, error) {
+func (o *Okcoin) GetDefaultConfig(ctx context.Context) (*config.Exchange, error) {
 	o.SetDefaults()
 	exchCfg := new(config.Exchange)
 	exchCfg.Name = o.Name
 	exchCfg.HTTPTimeout = exchange.DefaultHTTPTimeout
 	exchCfg.BaseCurrencies = o.BaseCurrencies
 
-	err := o.SetupDefaults(exchCfg)
+	err := o.Setup(exchCfg)
 	if err != nil {
 		return nil, err
 	}
-
 	if o.Features.Supports.RESTCapabilities.AutoPairUpdates {
 		err = o.UpdateTradablePairs(ctx, true)
 		if err != nil {
 			return nil, err
 		}
 	}
-
 	return exchCfg, nil
 }
 
 // SetDefaults method assigns the default values for OKCoin
-func (o *OKCoin) SetDefaults() {
+func (o *Okcoin) SetDefaults() {
```

Comment:
Is `Margin` supported by the websocket? If not, you'll need to add 
```go
	err = b.DisableAssetWebsocketSupport(asset.Margin)
	if err != nil {
		log.Errorln(log.ExchangeSys, err)
	}
```
So you don't get the following:
```
[WARN]  | SYNC | 14/07/2023 11:15:42 | okcoin USDC-USDT SPOT: No orderbook update after 1m0s, switching from websocket to rest
[WARN]  | SYNC | 14/07/2023 11:15:42 | okcoin BTC-USD MARGIN: No ticker update after 1m0s, switching from websocket to rest
[ERROR] | SYNC | 14/07/2023 11:15:42 | Failed to get REST ticker. Error: unsupported asset type, asset: margin
```

Reply:
Change applied, sir. Thanks for reminding me.